app/src/components/ProfileDropDown/ProfileDropDown.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Navbar, Nav, NavItem, NavDropdown, MenuItem, Image } from 'react-bootstrap';
+import { LinkContainer } from 'react-router-bootstrap';
+import { Link } from 'react-router';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as loginActions from '../../redux/Login';
+import * as profileActions from '../../redux/ProfileReducers';
+
+class ProfileDropDown extends React.Component {
+
+  componentWillMount() {
+    this.props.actions.loginActions.checkToken();
+    this.props.actions.profileActions.retrieveProfile(this.props.userId);
+    this.setState({profile: this.props.profile});
+  }
+
+  render() {
+    const profile = this.props.profile;
+    return (
+      <div >
+        <NavDropdown style={{display: 'inline-block'}} eventKey={5} id='basic-nav-dropdown'
+          title={profile.firstName + ' ' + profile.lastName}>
+          <LinkContainer to='/profile'>
+            <MenuItem eventKey={5.1}>Edit Profile</MenuItem>
+          </LinkContainer>
+          <MenuItem eventKey={5.2} onClick={this.props.actions.loginActions.logout}>Log Out</MenuItem>
+        </NavDropdown>
+        <span> 
+          <Image src={require('../../../img/profile.svg')} width='45'/>
```

Comment:
I think it will be better to use image as background to some element like div. In scss file:  `.yourdiv { background: url('data:image/svg+xml;utf8,<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" ......}`